### ISSUE TITLE
Undefined property mdl_payments_custom after posting a Payments Form

### ISSUE THREAD

**Issue by Bishawjit-Mondol** [InvoicePlane/InvoicePlane#1146]
## Expected Behavior
the amount will be saved in DB. 

## Current Behavior
![image](https://github.com/user-attachments/assets/94f224dc-324c-43a9-941a-c5bff268d650)

## Steps to Reproduce

1. https://demo.invoiceplane.com/payments/form/708
2. amount = 2000
3. bug will be generated

## Context (Environment)

Reason -> The amount is larger than due. (Check Please)

**Comment by nielsdrost7** [comment]
```
call to a member function get_by_pay_id on null
```
Payments (controller) line 115

**Comment by nielsdrost7** [comment]
Thank you for the big report, I'll take a look as soon as I can

**Comment by sudwebdesign** [comment]
Hi, After some search,
Throw an error **only** if posted amount > invoice amount

Bad idea, but solve this error:
In [Mdl_payments](https://github.com/InvoicePlane/InvoicePlane/blob/development/application/modules/payments/models/Mdl_payments.php#L110) line 110 replace `return false` by
```php
redirect('payments/form/' . $payment_id);
```
but after redirect, no message.
If help you.

**Comment by sudwebdesign** [comment]
Hi, it work normally and print user error message with #1151

(French msg error) `Le montant du paiement ne peux pas excéder le solde de la facture.`

**Comment by nielsdrost7** [comment]
@sudwebdesign thank you for that solution man!
Methee

**Comment by Bishawjit-Mondol** [comment]
Good job.

**Comment by nielsdrost7** [comment]
Solved by [releasing version 1.6.2](https://github.com/InvoicePlane/InvoicePlane/releases/tag/v1.6.2)
